### ISSUE TITLE
Read Screen Broken in v0.0.7

### ISSUE THREAD

**Issue by leesalminen** [boltcard/bolt-nfc-android-app#19]
Running Android 10, v0.0.7 of the `bolt-nfc-android-app` downloaded as an .apk from Github. 

When I open the app, navigate to Read NFC screen, and tap a card (a previously programmed card by this app), the UI does not update. The device beeps to indicate a NFC tag was read, but the app continues to say "Hold NFC card to Reader" with the spinner.

This worked in previous version (though I can't confirm exactly which one was working at this time).

**Comment by robertclarkson** [comment]
I'm afraid I cant see this issue happening, are there any steps to take to get it to break?

**Comment by leesalminen** [comment]
OK, I cloned down the repo and got a debug build of the app running on my device. In `MainActivity.java`, I added a logger in `onNewIntent` to spit out the value from `intent.getAction()`. I get `android.nfc.action.TAG_DISCOVERED` in my ADB logs. 


In `onNewIntent`, there's some code:

```
//if intent is not an NDEF discovery then do super and return;
if (!intent.getAction().equals("android.nfc.action.NDEF_DISCOVERED")) {
      super.onNewIntent(intent);
      return;
}
```

So, It looks like `readCard()` is never executed because my tag is showing an intent action of `android.nfc.action.TAG_DISCOVERED` instead of `android.nfc.action.NDEF_DISCOVERED`.

It looks like this bit of code was added in commit `c1ef22622c5dcdd45176fa19e1078394c21134ea`. 

Just for fun, I modified that code block to also allow execution on `android.nfc.action.TAG_DISCOVERED` intents. Code like this:

```
if (!intent.getAction().equals("android.nfc.action.NDEF_DISCOVERED") && !intent.getAction().equals("android.nfc.action.TAG_DISCOVERED")) {
      super.onNewIntent(intent);
      return;
}
```

Now, when I tap the card, the Read screen works as expected.

Happy to send a PR for this change if you'd like. Though, I'm unsure if there was a specific reason this code block was added.

**Comment by robertclarkson** [comment]
Thanks for the PR! Have merged this in v0.0.7.1
